soccer/gameplay/skills/pass_receive.py

```diff
@@ -261,9 +261,9 @@ def execute_receiving(self):
     def role_requirements(self):
         reqs = super().role_requirements()
         for req in role_assignment.iterate_role_requirements_tree_leaves(reqs):
-            if self._target_pos != None:
-                req.destination_shape = self._target_pos
-            elif self.receive_point != None:
+            #if self._target_pos != None:
+            #    req.destination_shape = self._target_pos
+            if self.receive_point != None:
```

Comment:
Does anyone remember why this is commented out? It looks like `_target_pos` updates to a new point after the ball is kicked and is really the one we should using here. `receive_point` is the initial target pass position.

Reply:
> Quick fix to remove the target pos role requirements
Target pos is causing problems in the first few frames due to the ball velocity being completely wrong due to the ball filter. This may add some inefficiency since the closest robot to the true ball pass location is not being used.

Thank you for actually writing good commit messages, [but I cant resist](https://i.redd.it/5z3kspwmpkbz.jpg)

9ca01b54f6130d127000bb80326a668cd2df27a2

Someone should probably just fix the ball filter, but you could do some sort of 'stabilization' here (only use this if it hasn't changed more than x in the last x frames).